packages/design-system/src/components/table/components/filtersSidebar/listItem.tsx

```diff
@@ -128,7 +129,9 @@ const ListItem = ({
               className="text-link ml-2 mt-1 block text-royal-blue dark:text-medium-persian-blue"
               href="#"
             >
-              {isExpanded ? 'Show Less' : 'Show More'}
+              {isExpanded
+                ? I18n.getMessage('showLess')
+                : I18n.getMessage('showMore')}
```

Comment:
Shorter alternative.

```suggestion
{I18n.getMessage(isExpanded ? 'showLess' : 'showMore')}
```